### PR TITLE
Add nis stub

### PR DESCRIPTION
**Pull request by ymyzk**
Add a new stub for `nis` module. ([Python 3 docs](https://docs.python.org/3/library/nis.html) / [Python 2 docs](https://docs.python.org/2/library/nis.html))

### CODE REVIEW

File: stdlib/2and3/nis.pyi [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Stubs for nis (Python 3.6)
```
Comment (gvanrossum) [inline stdlib/2and3/nis.pyi]: This first line is not very useful.

File: stdlib/2and3/nis.pyi [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+# Unix only
```
Comment (gvanrossum) [inline stdlib/2and3/nis.pyi]: This comment could be replaced by wrapping the rest of the code in
```py
if sys.platform != 'win32':
```

File: stdlib/2and3/nis.pyi (hunk 1 above)
Comment (ymyzk) [reply stdlib/2and3/nis.pyi]: OK, I remove this line.

File: stdlib/2and3/nis.pyi (hunk 2 above)
Comment (ymyzk) [reply stdlib/2and3/nis.pyi]: Thanks for your good advice!

**Pull request merged**